template.yml

```diff
@@ -262,6 +262,82 @@ Resources:
       Environment:
         Variables:
           AWS_LAMBDA_EXEC_WRAPPER: /opt/change_runtime_directory
+          OUTPUT_BUCKET_NAME: !Ref OutputBucket
+          GET_OUTPUT_URL: !Sub "https://${OutputDomain}"
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub  "cdo-dev-${SubDomainName}-output"
```

Comment:
Is there a way to conditionally use the prefix `cdo-` when BaseDomainName is `code.org` and `cdo-dev-` when BaseDomainName is `dev-code.org`?

Not sure that's possible with [CloudFormation Condition Functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html). If not, we may to make the S3 Bucket Name prefix a template Parameter or hard code the prefix to `cdo-` and hope that no-one tries to make an instance of Javabuilder in the Dev account that conflicts with one in the production account.

Reply:
Good catch, meant to fix this before my PR. It looks like I can do it conditionally